easybuild/cray_external_modules_metadata-20.05.cfg

```diff
@@ -0,0 +1,122 @@
+# metadata useful to EasyBuild (hpcugent.github.io/easybuild) for modules provided by Cray on Cray systems
+# authors: Guilherme Peretti-Pezzi (CSCS), Petar Forai (IMP/IMBA), Kenneth Hoste (HPC-UGent), Eirini Koutsaniti (CSCS)
+
+[cray-fftw]
```

Comment:
@ekouts, I think we can drop almost all variables from the metadata file.
What about
```
[cray-fftw]
name = FFTW

[cray-hdf5]
name = HDF5

[cray-hdf5-parallel]
name = HDF5

[cray-libsci]
name = LibSci

[cray-netcdf]
name = netCDF, netCDF-Fortran

[cray-netcdf-hdf5parallel]
name = netCDF, netCDF-Fortran

[cray-petsc]
name = PETSc

[cray-petsc-64]
name = PETSc

[cray-petsc-complex]
name = PETSc

[cray-petsc-complex-64]
name = PETSc

[cray-python/2.7.15.6]
name = Python
version = 2.7.15.6
prefix = /opt/python/2.7.15.6

[cray-python/2.7.15.7]
name = Python
version = 2.7.15.7
prefix = /opt/python/2.7.15.7

[cray-python/3.6.5.6]
name = Python
version = 3.6.5.6
prefix = /opt/python/3.6.5.6

[cray-python/3.6.5.7]
name = Python
version = 3.6.5.7
prefix = /opt/python/3.6.5.7

[cray-python/3.7.3.2]
name = Python
version = 3.7.3.2
prefix = /opt/python/3.7.3.2

[cray-python/3.8.2.0]
name = Python

[cray-R]
name = R

[cray-tpsl]
name = TPSL

[cray-tpsl-64]
name = TPSL

[cudatoolkit/10.1.105_3.27-7.0.1.1_4.1__ga311ce7]
name = CUDA
version = CRAY_CUDATOOLKIT_VERSION
prefix = CRAY_CUDATOOLKIT_DIR

[gcc]
name = GCC

[papi]
name = PAPI

[pmi]
name = pmi

[slurm/.default]
name = slurm
version = .default
prefix = SLURMDIR
```

Reply:
I think we still have to keep some of those, at least `cray-petsc-64`, `cray-petsc-complex`, `cray-petsc-complex-64` and `cray-tpsl-64`, but I will try it, no problem

Reply:
Removed all of them, except for the ones I mention in the comment above